lib/apiservers/engine/backends/volume.go

```diff
@@ -110,21 +109,24 @@ func (v *Volume) VolumeCreate(name, driverName string, opts, labels map[string]s
 		return result, derr.NewErrorWithStatusCode(fmt.Errorf("Bad Driver Arg: %s", varErr), http.StatusBadRequest)
 	}
 
-	// TODO: setup name randomization if name == nil
+	if model.Name == "" {
+		volumeUUID, _ := uuid.NewUUID()
+		model.Name = volumeUUID.String()
```

Comment:
Can we not use random uuid?  `uuid.New().String()`

Or does this need to be a version 1 uuid?